test/e2e/config/ironic.yaml

```diff
@@ -49,3 +60,5 @@ intervals:
   default/wait-deprovisioning: ["1m", "10ms"]
   default/wait-deleted: ["20s", "10ms"]
   default/wait-secret-deletion: ["1m", "1s"]
+  upgrade/wait-provisioning: ["1m", "1s"]
+  upgrade/wait-provisioned: ["10m", "1s"]
```

Comment:
Why not reuse `default/wait-provisioning` and `default/wait-provisioned`? There should not be a difference in the time required. I would rather use those existing values and update them if needed

Reply:
In my trials, there were a bunch of times these waits are not sufficient, so I added those. Maybe I should update the values to the default ones instead?